pkg/minikube/assets/addons.go

```diff
@@ -332,7 +333,7 @@ var Addons = map[string]*Addon{
 // AddMinikubeDirAssets adds all addons and files to the list
 // of files to be copied to the vm.
 func AddMinikubeDirAssets(assets *[]CopyableFile) error {
```

Comment:
@tstromberg 
I had moved `GuestXXX` constant, and change to read `vmpath` package. But It is failed...
```
golangci/golangci-lint info installed out/linters/golangci-lint
pkg/minikube/assets/addons.go:334:14: GuestAddonsDir not declared by package constants (typecheck)
			constants.GuestAddonsDir,
			          ^
```
and 
```
# k8s.io/minikube/pkg/minikube/assets [k8s.io/minikube/pkg/minikube/assets.test]
pkg/minikube/assets/addons.go:334: undefined: constants.GuestAddonsDir
pkg/minikube/assets/addons.go:340: undefined: constants.GuestAddonsDir
pkg/minikube/assets/addons.go:346: undefined: constants.GuestAddonsDir
```

Reply:
Sorry, I have misunderstood to have already rebased upstream..